src/overlays/actors/ovl_En_Po_Fusen/z_en_po_fusen.c

```diff
@@ -72,10 +72,10 @@ void EnPoFusen_Init(Actor* thisx, GlobalContext* globalCtx) {
 
     if (0) {}
     this->collider.dim.worldSphere.radius = 40;
-    SkelAnime_InitSV(globalCtx, &this->anime, &D_060024F0, &D_06000040, &this->limbDrawTbl, &this->transitionDrawTbl,
-                     10);
+    SkelAnime_InitSV(globalCtx, &this->anime, (SkeletonHeader*)&D_060024F0, (AnimationHeader*)&D_06000040,
```

Comment:
I feel like `D_060024F0` and `D_06000040` should be declared to their correct types instead of casting them like this.

Reply:
Ah yeah good catch. Turns out the types were just swapped.